### ISSUE TITLE
Abbreviated words

### ISSUE THREAD

**Issue by Daut** [blakeembrey/pluralize#49]
Hey,

I'm currently working on a project that needs to support pluralization of the abbreviated words, so I need to add lowercase 's' letter to the end of these words (e.g. VIP => VIPs, currently I get VIPS).

Are you planning to add this feature to your library? Something like `pluralize.addAbbreviatedRule()` or similar.

Regards

**Comment by blakeembrey** [comment]
I thought there had been a previous discussion around this sort of thing, but I can't recall it now. I think it's reasonable to want to do this, but in reality I'd need to know a lot more about the context to figure out if it's something that should be here. Can you give more information? An abbreviated rule is pretty similar to an irregular rule, what would be the main difference? Is it just how to handle casing, in which case is there perhaps a better way (maybe a case-sensitivity flag?) that doesn't change the core behaviour?

**Comment by Daut** [comment]
Yes, In my use case only difference between an abbreviated rule and irregular rule is case handling. So something like `addIrregularRule(single, plural, preserveCase)` will do the trick for me.

**Comment by davidcalhoun** [comment]
Just ran into the same issue - also with the string `id`.

Example:
```js
pluralize('Foo ID', 2);  // -> 'Foo IDS', but I instead want to output 'Foo IDs'
```
